arbor/fvm_layout.hpp

```diff
@@ -110,6 +116,8 @@ struct fvm_mechanism_config {
 
     // (Non-global) parameters and parameter values across the mechanism instance.
     std::vector<std::pair<std::string, std::vector<value_type>>> param_values;
+
+    bool linear;
```

Comment:
This was used to set the layout.coalesced_synapses flag in fvm_lowered_cell_impl.hpp. However, after adopting the convention that an empty multiplicity vector implies no coalesced synapses, this can be removed